### PR TITLE
#id 35523 | update bid entity

### PR DESCRIPTION
**Pull request by dave101ua**
- loaded switched from boolean to "Yes" / "No"
- skore switched from array of numbers to array of strings

### CODE REVIEW

**Review by Neofox — APPROVED** [review]

File: test/bid.test.js [hunk 1]
```diff
@@ -59,6 +59,36 @@ describe('Bid object structure', () => {
...
+    const [err0, val0] = s.validate(payload, BidStruct, {
+      coerce: true, mask: true
+    })
+    expect(err0).toBeUndefined()
+    expect(val0).toBeDefined()
+  })
+
+  test('Success: Bid structure with Yes as loaded', () => {
+    let payload = JSON.parse(JSON.stringify(Bids[0]));
+    payload.loaded = "No";
```
Comment (fanshan) [inline test/bid.test.js:74]:
```suggestion
    payload.loaded = "Yes";
```

File: structures/bid.js [hunk 2]
```diff
@@ -48,10 +48,10 @@ exports.bid = function(config = null, auction = null) {
       lang: s.optional(s.string()),
       forwarder: s.optional(s.string()),
       decision_maker: s.optional(s.string()),
-      loaded: s.optional(s.boolean()),
+      loaded: s.optional(s.enums(["Yes", "No"])),
       tracker: s.optional(s.string()),
       driver: s.optional(s.string()),
-      score: s.optional(s.array(s.number())),
+      score: s.optional(s.array(s.string())),
```
Comment (fanshan) [inline structures/bid.js:54]: Breaking change... What about an array of string OR number ?

**Review by fanshan — CHANGES_REQUESTED** [review]

File: structures/bid.js (hunk 2 above)
Comment (dave101ua) [reply structures/bid.js:54]: Increase major version of tag ?

Comment (fanshan) [reply structures/bid.js:54]: No

**Pull request merged**